build.gradle

```diff
@@ -28,7 +28,7 @@ ext.puppycrawlCheckstyleVersion = project.puppycrawlCheckstyleVersion
 ext.slf4jVersion = "1.7.30"
 ext.natsVersion = "2.8.0"
 ext.natsStreamingVersion = "2.2.3"
-ext.protobufVersion = "3.20.1"
+ext.protobufVersion = "3.20.3"
```

Comment:
Better to move these versions to the gradle.properties file. Isn't it hard to maintain multiple files for maintaining versions?

Reply:
This module is actually de-prioritised in every way. I've spent almost a day of JetStream time on this already. We will anyway have to deprecate this after JetStream. Shall we do this some other time 😅. Will do this if I have some time while doing JetStream. I'll create an issue. Is that okay.

Reply:
+1